packages/story-editor/src/components/panels/design/animation/animation.js

```diff
@@ -263,7 +264,7 @@ function AnimationPanel({
             disabledTypeOptionsMap={disabledTypeOptionsMap}
             direction={getEffectDirection(updatedAnimations[0])}
             selectedEffectType={updatedAnimations[0]?.type}
-            selectButtonStylesOverride={highlight?.focus && styles.OUTLINE}
+            selectButtonStylesOverride={highlight?.focus && OUTLINE}
```

Comment:
Curious why this change? Moving styles separately.

Reply:
Style.js not converted. So I had to side step this problem by loading it into a different import.